gui/widgets/scaledview.cpp

```diff
@@ -12,13 +12,14 @@
 #include <QGLWidget>
 #include <QPainter>
 #include <QGraphicsSceneEvent>
+#include <QDebug>
 
 #include <iostream>
 
 /* TODO: do we really want sample buffers for these views? configurable?
  */
 ScaledView::ScaledView()
-	: width(50), height(50) // values don't matter much, but should be over 0
+    : width(50), height(50), zoom(1), sm(Zoom)// values don't matter much, but should be over 0
```

Comment:
Why don't they matter? Don't they need to be consistent with the GUI? Maybe the Zoom etc buttons need to be ToggleButtons?